### DISCUSSION TITLE
docker-compose.yml with rspamd (discontinued)

### DISCUSSION THREAD

**Discussion by MyUncleSam** [simple-login/app#2089] [Self-hosting question]
Hi there,

at the moment I am using another service like simplelogin but I am thinking on moving to simplelogin. As I really like the rspamd and docker compose setup I wanted to share my setup. The rspamd part is just by flying over the documentation. So if there is a rspamd pro it would be nice to habe another opinion :-)

### use at your own risk
Title says everything: use at your own risk

### WIP - work in progress
This was built on a test system which is fully operational. I did this to have the possibility to compare it to the system I am currently using. So this could grow or die - depending if I continue using it :-)

### known stuff
- rspamd looses the history after a restart
- rspamd checks also outgoing mails which always gets a forged recipients point, something like self whitelisting would be great here
- the rspamd folder rspamd/db needs to get 777 permissions (chmod 777 rspamd/db)
- I am not using th sl-netwotk - I am using Internal, so change this if it bothers you (also 172.20.0.0/16 instead of 10.0.0.0/8)
- the migration runs to initialize the database or to upgrade betweens versions are not part of the compose file

### docker-compose.yml
[docker-compose.yml.txt](https://github.com/simple-login/app/files/15053351/docker-compose.yml.txt)

### rspamd
I attach the configuration of the rspamd I did. Keep in mind to generate a password for the webui or delete the worker-controller.inc file. To create a ui password use: docker run --rm rspamd/rspamd rspamadm pw -q -p HelloWorld

[conf.zip](https://github.com/simple-login/app/files/15053323/conf.zip)

**Comment by loeffelpan** [comment]
Sorry for bothering you, but I'm about to run lastest version of `app-ci` image on my homeserver to use the iOS app with that instance.
I saw that you're using `v4.42.1` in your compose file (which is way more up to date than "latest" `v4.6.5-beta`).

I have the same problem as discussed in #2107 and #2108 when I follow [these](https://github.com/simple-login/app/blob/new-self-host-version/docs/upgrade.md) instructions to run `app-ci` image. `v4.6.5-beta` is running fine with above instructions.
Can you please give some information how to run that image?

**Reply by MyUncleSam** [reply]
I was writing this after I successfully set up simplelogin for tests. It is a great system but lacks some core functions and features so I switched to addy.io which is working fine for me. As I do no longer have configuration files I cannot tell you how to proceed here.

By searching for the documentation I used I found this article. Maybe it helps: https://github.com/simple-login/app/discussions/1917#discussioncomment-7891289

**Comment by MyUncleSam** [comment]
As I am no longer using simplelogin I am closing this discussion.
